WallPaperDev/WallPaperDev/Networking/NetworkingService.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  NetworkingService.swift
+//  WallPaperDev
+//
+//  Created by Jamar Gibbs on 10/30/19.
+//  Copyright © 2019 Stephen Ouyang. All rights reserved.
+//
+
+import Foundation
+
+// TODO: Clean this up and create temp Key.swift file for Unsplash API key
+// TODO: Create a handler for no internet connection
+
+class NetworkingService {
+  
+  static let shared = NetworkingService()
+  private init() {}
+  
+  func getData(parameters: [String : String], completion: @escaping (Data) -> () ) {
+    
+    let baseURL = "https://api.unsplash.com/search/photos"
+    var newURL = URLComponents(string: baseURL)
+    newURL?.queryItems = []
+    
+    for param in parameters {
+      newURL?.queryItems?.append(URLQueryItem(name: param.key, value: param.value))
+    }
+    
+    guard let url = newURL?.url else {
+      #if DEBUG
+        print("invalid url")
+      #endif
+      return
+    }
+    
+    var request = URLRequest(url: url)
+//        request.addValue("Bearer \(APIKey.key)", forHTTPHeaderField: "Authorization")
```

Comment:
By not having an auth token what are the risks right now?

Reply:
I am removing this file in favor of the new network layer.